Week-1/index.html

```diff
@@ -0,0 +1,219 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
Add all the minimum relevant Meta tags 

meta charset="UTF-8"
meta name="referrer" content="origin-when-cross-origin" />
meta name="viewport" content="width=device-width, initial-scale=1.0">
meta http-equiv="X-UA-Compatible" content="IE=edge">
meta name="description" content="Awwwards are the Website Awards that recognize and promote the talent and effort of the best developers, designers and web agencies in the world.">
meta name="theme-color" content="#3ea094">
meta name="keywords" content="Website Awards, Web Design Inspiration, Webdesign Trends" >
meta property="og:site_name" content="Awwwards">
meta property="og:image" content="https://assets.awwwards.com/assets/images/pages/about-certificates/awwwards.jpg" />
meta property="og:title" content="Awwwards - Website Awards - Best Web Design Trends" />
meta property="og:type" content="website" />
meta property="og:url" content="https://www.awwwards.com/" />
meta property="og:description" content="Awwwards are the Website Awards that recognize and promote the talent and effort of the best developers, designers and web agencies in the world." />